pkgs/by-name/bs/bs-manager/package.nix

```
@@ -0,0 +1,172 @@
+{
+  alsa-lib,
+  autoPatchelfHook,
+  atk,
+  bash,
+  cairo,
+  cups,
+  dbus,
+  dpkg,
+  fetchurl,
+  ffmpeg,
+  glib,
+  glibc,
+  gtk3,
+  lib,
+  libappindicator-gtk3,
+  libdrm,
+  libxkbfile,
+  libxkbcommon,
+  makeWrapper,
+  mesa,
+  nss,
+  nspr,
+  nix-update-script,
+  pango,
+  pkgs,
+  python3,
+  opencomposite,
+  stdenv,
+  systemd,
+  vulkan-loader,
+  xorg,
+  depotdownloader,
+  openssl,
+  wrapGAppsHook3,
+}:
+let
+  version = "1.5.0-alpha.8";
+
+  src =
+    if stdenv.hostPlatform.system == "x86_64-linux" then
+      fetchurl {
+        url = "https://github.com/Zagrios/bs-manager/releases/download/v1.5.0-alpha.8/bs-manager_1.5.0-alpha.8_amd64.deb";
+        hash = "sha256-xQ3ha6J1wqzZj25QwlMX2lwDDNXlzVvpUqXkaH9kCbE=";
+      }
+    else
+      throw "BSManager is not available for your platform";
+
+  rpath = lib.makeLibraryPath [
+    alsa-lib
+    atk
+    cairo
+    cups
+    dbus
+    ffmpeg
+    glib
+    glibc
+    gtk3
+    libappindicator-gtk3
+    libdrm
+    libxkbcommon
+    libxkbfile
+    mesa
+    nspr
+    nss
+    pango
+    stdenv.cc.cc
+    vulkan-loader
+    xorg.libX11
+    xorg.libXcomposite
+    xorg.libXcursor
+    xorg.libXdamage
+    xorg.libXext
+    xorg.libXfixes
+    xorg.libXi
+    xorg.libXrandr
+    xorg.libXrender
+    xorg.libXScrnSaver
+    xorg.libXtst
+    xorg.libxcb
+    xorg.libxkbfile
+  ];
+in
+stdenv.mkDerivation {
+  pname = "bs-manager";
+  inherit version;
+
+  system = "x86_64-linux";
+
+  inherit src;
+
+  runtimeDependencies = [
+    bash
+    opencomposite
+    python3
+    systemd
+    depotdownloader
+    openssl
+  ];
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    dpkg
+    ffmpeg
+    mesa
+    nspr
+    nss
+    makeWrapper
+    openssl
+    wrapGAppsHook3
+  ];
+
+  buildInputs = [
+    ffmpeg
+    openssl
```

```suggestion
    openssl
    libgbm
    nspr
    nss
```
These were previously taken out of the `nativeBuildInputs`. It is better to put them here. `nss`, `nspr` in particular can probably be removed from nativeBuildInputs too.

`libgbm` is preferred over `mesa` whenever possible, try replacing it elsewhere too.

```
error: auto-patchelf could not satisfy dependency libnss3.so wanted by /nix/store/g7jw74h8crzngch0mg6fkv1ngqw8z5ri-bs-manager-1.5.0-alpha.8/opt/BSManager/bs-manager
error: auto-patchelf could not satisfy dependency libnssutil3.so wanted by /nix/store/g7jw74h8crzngch0mg6fkv1ngqw8z5ri-bs-manager-1.5.0-alpha.8/opt/BSManager/bs-manager
error: auto-patchelf could not satisfy dependency libsmime3.so wanted by /nix/store/g7jw74h8crzngch0mg6fkv1ngqw8z5ri-bs-manager-1.5.0-alpha.8/opt/BSManager/bs-manager
error: auto-patchelf could not satisfy dependency libnspr4.so wanted by /nix/store/g7jw74h8crzngch0mg6fkv1ngqw8z5ri-bs-manager-1.5.0-alpha.8/opt/BSManager/bs-manager
error: auto-patchelf could not satisfy dependency libgbm.so.1 wanted by /nix/store/g7jw74h8crzngch0mg6fkv1ngqw8z5ri-bs-manager-1.5.0-alpha.8/opt/BSManager/bs-manager
```